[IMP] Time Off: updating requested carry over time info (#10477)
odoo/documentation: content/applications/hr/time_off.rst

:ref:`public holidays <time_off/public-holidays>` can be set.

.. note::
   Be advised, only users with specific access rights can see all aspects of the **Time Off** app.

   All users can access the *My Time Off*  (:menuselection:`Time Off app --> My Time --> My Time

>    Off`) and *Overview* (:menuselection:`Time Off app --> Overview`) sections of the **Time Off**

Reviewer: I don't like this, we're creeping back into strange use of parenthesis and overall let's stop adding navigational instructions when they're not part of an active instruction.

cc: @ksc-odoo

   app. All other sections require specific access rights.

>    To better understand how access rights affect the **Time Off** app, refer to the

Reviewer: For reference, the word "application" is fine. You don't need to say or switch to the word "app" wholeheartedly.

To view the currently configured time off types, navigate to :menuselection:`Time Off app -->
Configuration --> Time Off Types`. The time off types are presented in a list view.

> The **Time Off** app comes with four pre-configured time off types: :guilabel:`Paid Time Off`,

Reviewer 2: No hyphen please in "preconfigured"
```suggestion
The **Time Off** app comes with four preconfigured time off types: :guilabel:`Paid Time Off`,
```

    upcoming year.
  - :guilabel:`At the allocation date`: Select this if the accrual rolls over as soon as time is
    allocated to the employee.
  - :guilabel:`Other`: Select this option if neither of the other two options are applicable. When
    selected, a :guilabel:`Carry-Over Date` field appears. Select the date using the two drop-down
    menus, one for the day and one for the month.

- :guilabel:`Based on worked time`: Enable this option if time off accrual is determined by the
  employee's worked hours. Days **not** considered as worked time do **not** contribute to the
  accrual plan in Odoo.
- :guilabel:`Milestone Transition`: This field is **only** visible after a minimum of two
  :ref:`rules <time_off/rules>` have been configured on the accrual plan. This selection determines
  when employees move up to a new milestone. If they qualify to change milestones in the middle of a
  pay period, decide whether the employee changes milestones :guilabel:`Immediately` or
  :guilabel:`After this accrual's period` (after the current pay period).
- :guilabel:`Company`: Using the drop-down menu, select the company the accrual plan applies to. If

>   left blank, the accrual plan can be used for all companies. This field **only** appears in a

Reviewer: Qualifying context like "this field only appears in a multi-company database" should be read first before the field description in order to save the reader time on what's relevant for them in the doc.

  the employee starts to accumulate time off. The first value is numerical; enter a number in the
  first field.

  Then, select the type of time period using the drop-down menu in the second field. The options
  are: :guilabel:`Days`, :guilabel:`Months`, or :guilabel:`Years`.
- :guilabel:`Carry over`: select how any unused time off is handled. The options are either:

  - :guilabel:`None. Accrued time reset to 0`: Any unused time off is gone.
  - :guilabel:`All accrued time carried over`: All unused time off is rolled over to the next
    calendar year.
  - :guilabel:`Carry over with a maximum`: Unused time off is rolled over to the next calendar year,
    but there is a cap. An :guilabel:`Up to` field appears if this is selected. Enter the maximum
    number of :guilabel:`Days` that can roll over to the following year. Any time off beyond this
    parameter is lost.

.. important::
   If the :guilabel:`Carry over` field is set to :guilabel:`None. Accrued time reset to 0`, that
   rule *overrides* the :guilabel:`Carry-Over Time` set on the accrual plan.

   If a company creates an accrual plan, granting employees time off :guilabel:`At the start of the

>    accrual period` (the beginning of the year), and sets the :guilabel:`Carry-Over Time` on the

Reviewer: ```suggestion
   accrual period` (i.e., the beginning of the year), and sets the :guilabel:`Carry-Over Time` on the
```

Reviewer: Is this even needed since the accrual plan is set to this period, which has "start of the year" in the label?

Author: Yes, since the rules determine when the accrual period starts - it could be different, so I wanted to be clear that both were set to the beginning of the year.

   *accrual plan* to :guilabel:`At the start of the year`, it allows unused vacation time to
   rollover to the following year.

   Then, the company adds rules to the accrual plan, allocating five days of vacation, annually, on
   the first of the year (one week of vacation allocated on January 1st).

   If the :guilabel:`Carry over` field is set to :guilabel:`None. Accrual time reset to 0` on the

>    :guilabel:`Create Milestone` pop-up for, any unused vacation time **DOES NOT carry over**, even

Reviewer: subjective formatting, italics is fine (no bold/caps).
the consequence of what happens here is implied by the label, if we really want to make it a big deal use an admonition block (warning/important)

```suggestion
   :guilabel:`Create Milestone` pop-up for, any unused vacation time *does not* carry over, even
```

   though on the :guilabel:`Accrual Plan` form, the :guilabel:`Carry-Over Time` is set to
   :guilabel:`At the start of the year`.

>    The carry over set on the *rule* **takes precedence** over the carry over set on the *accrual

Reviewer: not necessary, we said it plainly
```suggestion
   The carry over set on the *rule* takes precedence over the carry over set on the *accrual
```

Enter the following information on that new line:

- :guilabel:`Name`: Enter the name of the holiday.
- :guilabel:`Company`: If in a multi-company database, the current company populates this field by
  default. It is **not** possible to edit this field.

  .. note::
     The :guilabel:`Company` field is hidden, by default. To view this field, click the

>      :icon:`oi-settings-adjust` :guilabel:`optional columns` icon in the top-right corner of the

Reviewer 2: Preferred Icon label for this icon :)
```suggestion
     :icon:`oi-settings-adjust` :guilabel:`(additional options)` icon in the top-right corner of the
```

Enter the following information on that new line:

- :guilabel:`Name`: Enter the name of the mandatory day.
- :guilabel:`Company`: If in a multi-company database, this field is visible, and the current
  company populates this field, by default. Using the drop-down menu, select the company the
  mandatory day is for.
- :guilabel:`Departments`: This column is hidden by default. First, click the

>   :icon:`oi-settings-adjust` :guilabel:`optional columns` icon in the top-right corner, next to

Reviewer 2: ```suggestion
  :icon:`oi-settings-adjust` :guilabel:`(additional options)` icon in the top-right corner, next to
```

time off. This allows users to see which employees are taking time off, how much time off they are
taking, and what time off types are being used.

> Any report can be added to a spreadsheet, when in either the :icon:`fa-area-chart` :guilabel:`Graph`
> or :icon:`oi-view-pivot` :guilabel:`Pivot` view, through the *Insert in Spreadsheet* button that
> appears in the top-left of the report.

Reviewer 2: icon label formatting– needs parenthesis 
```suggestion
Any report can be added to a spreadsheet, when in either the :icon:`fa-area-chart`
:guilabel:`(Graph)` or :icon:`oi-view-pivot` :guilabel:`(Pivot)` view, through the *Insert in
Spreadsheet* button that appears in the top-left of the report.
```

taking, and what time off types are being used.

> Any report can be added to a spreadsheet through the *Insert in Spreadsheet* button that appears in

Reviewer 3: I am only seeing this button appear on the 'by Type' reporting page... it's definitely worth a double-check to ensure the accuracy of this statement during your revisions. Thanks! 👍

Author: It appears for the 'by Employee' report if you change to a graph view- I'll add that!

The report can be displayed in other ways, as well. Click the corresponding button option in the
top-right corner of the page to view the data in that specific way. The various options are a

> :icon:`oi-view-list` :guilabel:`List` (list), or default view, :icon:`fa-area-chart`
> :guilabel:`Graph`, :icon:`oi-view-pivot` :guilabel:`Pivot` table, or :icon:`fa-calendar`
> :guilabel:`Calendar` view.

Reviewer 2: ```suggestion
:icon:`oi-view-list` :guilabel:`(List)`, :icon:`fa-area-chart` :guilabel:`(Graph)`,
:icon:`oi-view-pivot` :guilabel:`(Pivot)`, or :icon:`fa-calendar` :guilabel:`(Calendar)` view.
```

The report can be displayed in other ways, as well. Click the corresponding button option in the
top-right corner of the page to view the data in that specific way. The various options are a

> :icon:`oi-view-list` :guilabel:`List` (the default view), :icon:`fa-area-chart` :guilabel:`Graph`,

Reviewer: fallback labels are only for that purpose

```suggestion
:icon:`oi-view-list` :guilabel:`List` (list), or default view, :icon:`fa-area-chart` :guilabel:`Graph`,
```

:guilabel:`List` (the default view), :guilabel:`Graph`, :guilabel:`Pivot` table, or

> :guilabel:`Calendar` view.

Reviewer 3: I'd consider adding the appropriate icons here, as well... just for a little added "visual value"

The report can be displayed in other ways, as well. Click the corresponding button option in the
top-right corner of the page to view the data in that way. The various options are a

> :icon:`fa-area-chart` :guilabel:`Graph` (the default view), :icon:`oi-view-list` :guilabel:`List`,
> or :icon:`oi-view-pivot` :guilabel:`Pivot` table.

Reviewer 2: ```suggestion
:icon:`fa-area-chart` :guilabel:`(Graph)` (the default view), :icon:`oi-view-list`
:guilabel:`(List)`, or :icon:`oi-view-pivot` :guilabel:`(Pivot)` table.
```
